framework/src/modules/validators/endpoint.ts

```diff
@@ -47,4 +53,34 @@ export class ValidatorsEndpoint extends BaseEndpoint {
 			valid: bls.popVerify(Buffer.from(blsKey, 'hex'), Buffer.from(proofOfPossession, 'hex')),
 		};
 	}
+
+	public async getValidator(
+		ctx: ModuleEndpointContext,
+	): Promise<{ generatorKey: string | undefined; blsKey: string | undefined }> {
+		validator.validate<GetValidatorRequest>(getValidatorRequestSchema, ctx.params);
+		const req = ctx.params;
+		const { address } = req;
+
+		const validatorsKeysSubStore = this.stores.get(ValidatorKeysStore);
+
+		let persistedValue;
+
+		try {
+			persistedValue = await validatorsKeysSubStore.get(ctx, Buffer.from(address, 'hex'));
+		} catch (error) {
+			if (!(error instanceof NotFoundError)) {
+				throw error;
+			}
+
+			return {
+				generatorKey: undefined,
+				blsKey: undefined,
+			};
```

Comment:
we can leave it as empty string,
```suggestion
			return {
				generatorKey: '',
				blsKey: '',
			};
```